src/pages/Import.js

```diff
@@ -82,7 +82,7 @@ class DockstoreImporter extends Component {
           mutabilityWarning
         ]),
         h(Collapse, { title: 'REVIEW WDL' },
-          [h(WDLViewer, { wdl })]
+          [h(WDLViewer, { wdl, style: { maxHeight: 300 } })]
```

Comment:
Shouldn't I get more height if I, for example, use my display in portrait orientation?